draft-ietf-ice-pac.xml

```diff
@@ -116,56 +116,56 @@
-          However, there are certain specific cases where this problem will
-          frequently occur.
+          In most ICE scenarios, the lengthy transaction timeouts for connectivity check
+          Binding requests, typically tens of seconds, will prevent 
+          this problem for occuring. However, there are certain specific cases 
+          where this problem will frequently occur.
         </t>
         <section title="No Candidates From Peer">
           <t>
-            It is entirely legal for an ICE gent to provide zero candidates of
-            its own. If the agent somehow knows that the remote endpoint is
+            It is entirely legal for an ICE agent to provide zero candidates of
+            its own. If the agent somehow knows that the peer agent is
             directly reachable, gathering local candidates is unnecessary and
-            will only cause delays; the remote endpoint can discover the
+            will only cause delays; the peer agent can discover the
             appropriate local candidate via connectivity checks.
           </t>
           <t>
             However, following the procedures from
             <xref target="RFC8445"></xref> strictly will result in immediate
-            ICE failure, since the checklist at the remote endpoint will be
+            ICE failure, since the checklist at the peer agent will be
             empty.
           </t>
         </section>
         <section title="All Candidates Discarded">
           <t>
             Even if the ICE agent provides candidates, they may be discarded
-            by the remote endpoint if it does not know what to do with them.
-            For example, candidates may use an address family that the remote
-            endpoint does not support, (e.g., a host candidate with an IPv6
+            by the peer agent if it does not know what to do with them.
+            For example, candidates may use an address family that the peer
+            agent does not support, (e.g., a host candidate with an IPv6
             address in a NAT64 scenario), or may not be usable for some other
             reason (e.g., a candidate that contains a FQDN that fails to
             resolve).
           </t>
           <t>
             In these scenarios, when the candidates are discarded, the
-            checklist at the remote endpoint will once again be empty, leading
+            checklist at the peer agent will once again be empty, leading
             to immediate ICE failure.
           </t>
         </section>
         <section title="Immediate Candidate Pair Failure">
           <t>
             Section 7.2.5.2 of <xref target="RFC8445"></xref> describes several
             situations in which a candidate pair will be considered to have
-            failed, well before the connectivity check timeout.
+            failed, well before the connectivity check Binding request transaction timeout.
```

Comment:
"connectivity check transaction timeout"

Reply:
I can do that.